packages/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -133,78 +140,93 @@ export class AreaChartBase extends React.Component<IAreaChartProps, IAreaChartSt
     }
   }
 
+  public componentDidMount(): void {
+    if (this._isChartEmpty) {
+      d3Select('body')
+        .append('div')
+        .attr('role', 'alert')
+        .attr('id', 'ariaLabel_AreaChart')
+        .style('opacity', 0)
+        .attr('aria-label', 'Graph has no data to display')
+        .attr('tabIndex', 0);
+    }
+  }
+
   public render(): JSX.Element {
-    const { lineChartData, chartTitle } = this.props.data;
-    const { colors, opacity, stackedInfo, calloutPoints } = this._createSet(this.props.data);
-    this._calloutPoints = calloutPoints;
-    const isXAxisDateType = getXAxisType(lineChartData!);
-    this._colors = colors;
-    this._opacity = opacity;
-    this._stackedData = stackedInfo.stackedData;
-    const legends: JSX.Element = this._getLegendData(this.props.theme!.palette, lineChartData!);
-
-    const tickParams = {
-      tickValues: this.props.tickValues,
-      tickFormat: this.props.tickFormat,
-    };
+    if (!this._isChartEmpty) {
```

Comment:
what is the render behavior if chart is empty

Reply:
it will return <></>;

Reply:
Can you show how the canvas looks in this case.

Reply:
already discussed offline